lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -217,13 +230,16 @@ def _get_lti_embed_code(self) -> str:
         return lti_embed(
             html_element_id='lti-tab-launcher',
             lti_consumer=self.configuration.lti_configuration.get_lti_consumer(),
-            resource_link_id=resource_link_id,
-            user_id=str(self.request.user.id),
+            resource_link_id=quote(resource_link_id),
+            user_id=quote(anonymous_id_for_user(self.request.user, None)),
             roles=self.get_user_roles(),
-            context_id=self.program_uuid,
-            context_title=self.program_uuid,
+            context_id=quote(self.program_uuid),
+            context_title=self._get_context_title(),
             context_label=self.program_uuid,
-            result_sourcedid=result_sourcedid
+            result_sourcedid=quote(result_sourcedid),
+            locale=to_locale(get_language()),
+            additional_params=self._get_additional_lti_parameters()
+
```

Comment:
Don't you want to add pii info here as well?

Reply:
I am not sure if it is in scope of this issue and in ACs of the ticket because to adding `pii_params`  is dependent on  [this model ](https://github.com/edx/xblock-lti-consumer/blob/1b1f3805bd293206a3100e9e01256ee7ced644a4/lti_consumer/models.py#L555)and currently it is course only model .